vhdbuilder/packer/components.json

```diff
@@ -193,15 +193,24 @@
       "downloadURL": "mcr.microsoft.com/oss/cilium/operator:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "1.10.5"
+        "1.10.5",
```

Comment:
I just saw a comment on your rp PR, could we remove the old versions since they're on being used anywhere?

Reply:
I left the old versions in because I was worried removing them might cause other PRs to fail the VHD preload check. Was planning to remove them once the other PR merges switching to the new versions. But maybe it's okay to remove them here?

Reply:
discussed offline, we'll add the old versions to the allow-list so the VHD preload continues to pass. Updated this PR to remove the old versions.